04/app.js

```diff
@@ -2,8 +2,26 @@ const user = {
     firstName: 'Adam',
     lastName: 'Nowak',
     born: {
-        day: '14',
-        month: '04',
-        year: '1985'
+        day: '27',
+        month: '03',
+        year: '2022'
     }
-}
\ No newline at end of file
+};
+
+const checkUserBirthday = function ({born}) {
+    const dateNow = new Date().toLocaleDateString().split('.');
```

Comment:
Należy uważać na wykorzystanie `[toLocaleDateString()](https://developer.mozilla.org/pl/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleDateString#przyk.c5.82ad_zastosowanie_tolocaledatestring)` ponieważ ten format będzie różnie wyglądał w zależności od lokalnego czasu. Co będzie powodować, że nie zawsze to prawidłowo zadziała bo w np. Polsce inaczej zapisujemy datę, a w USA czy EN inaczej.